benchmarks/glob_rule_matches/config.yaml

```diff
@@ -0,0 +1,19 @@
+# Every iteration should match, high cache hit rate
+high_match:
+  num_iterations: 100
+  num_strings: 1
+
+# Low number of variations, hit rate of about 25%
+average_match:
+  num_iterations: 100
+  num_strings: 2
+
+# High number of variations, hit rate of 0% or 1%
+low_match:
+  num_iterations: 100
+  num_strings: 25
+
+# This variation has performance issues due to the cache max size
```

Comment:
is this a realistic scenario, if it causes performance issues does that mean we'll introduce performance issues for customers or is this not a realistic/useful scenario?

Reply:
It's not very realistic. I'll remove.